awx/ui/client/src/templates/workflows/edit-workflow/workflow-edit.controller.js

```diff
@@ -261,11 +255,39 @@ export default [
             .map(i => ({id: i.id + "",
                 test: i.name}));
 
+        // Select2-ify the lables input
         CreateSelect2({
+            scope: $scope,
             element:'#workflow_job_template_labels',
             multiple: true,
             addNew: true,
-            opts: opts
+            opts,
+            callback: 'select2Loaded'
+        });
+
+        $scope.$on('select2Loaded', () => {
```

Comment:
https://github.com/ansible/awx/pull/3260/files#r258623541 - same thing in this file.  If we do keep the callback paradigm I don't think you need `select2Loaded`.  You can just use `select2LoadFinished` since there's only one select2 in this form.